frontend/src/metabase/dashboard/components/DashCard/DashCardVisualization.tsx

```diff
@@ -63,7 +63,6 @@ interface DashCardVisualizationProps {
 
   expectedDuration: number;
   isSlow: CardSlownessStatus;
-  isAction: boolean;
```

Comment:
`isAction` was never used.

Reply:
it looks like you're right, but I'd double check it

Reply:
It was used [some time ago](https://github.com/metabase/metabase/pull/26676/files). But I never bother to find when we stop using it. But it's not currently being used for sure. There's currently no one using `isAction` prop (apart from ones being removed in this PR).